api/e2e/test/00_all_e2e_test.go

```diff
@@ -16,9 +16,9 @@ import (
 const (
 	deploymentWaitTimeoutSeconds = 200
 	deleteTimeoutSeconds         = 20
-	fluentdSyncIntervalSeconds   = 10
+	fluentdSyncIntervalSeconds   = 10 // nolint:deadcode,varcheck,unused
```

Comment:
Since these vars are not used in the OSS version of the tests, do you think we can just remove them?

Reply:
OK I'll remove it